datafusion/core/tests/sqllogictests/test_files/aggregate.slt

```diff
@@ -1387,7 +1387,7 @@ as values
  ('2018-11-13T17:11:10.011375885995', 'Row 0'),
  ('2011-12-13T11:13:10.12345', 'Row 1'),
  (null, 'Row 2'),
- ('2021-1-1T05:11:10.432', 'Row 3');
+ ('2021-01-01T05:11:10.432', 'Row 3');
```

Comment:
Filed https://github.com/apache/arrow-rs/issues/3969 to track (I don't think we necessarily need to change arrow to support these timestamps, it but it will reduce confusion to have the change documented)